action.yml

```diff
@@ -32,6 +32,24 @@ inputs:
       for the surest of choices.
     required: false
     default: Changelog
+  versions-dropdownlist-configuration:
+    description:
+      YAML configuration how should the dropdownlist appear.
+      String should be quoted in ' sign
+    required: false
+    default: |
+      list(config = list(
+        tooltip = list(
+          devel = "Unreleased development version of Package", 
+          main = "CRAN Released Package v0.7.0"
+        ), 
+        text = list(
+          devel = "devel - unreleased", 
+          main = "main - CRAN"
+        )
+      ))
```

Comment:
This should not be the default. Default should be nothing.

Reply:
Done